README.md

```diff
@@ -16,16 +16,30 @@ The objective (not all these are yet implemented):
 
 ## Building
 
-First install Postgres. Once installed, this environment variable is required:
+First install Postgres. The build should be able to find the directory for the postgres server headers, it uses the `pg_config --includedir-server` to attempt to find the directory. If it is unsuccessful then this environment variable is required:
 
-`PG_INCLUDE_PATH=[/path/to/postgres]/include/server # e.g. /usr/local/pgsql/include/server`
+`PG_INCLUDE_PATH=/path/to/postgres]/include/server # e.g. /usr/local/pgsql/include/server`
```

Comment:
Looks like a typo, if you remove `[` you should remove `]` too :)

Reply:
Thanks, I noticed that one too. Not sure what happened there.